test/behavior/bitreverse.zig

```diff
@@ -158,7 +158,6 @@ test "bitReverse vectors u0" {
     if (builtin.zig_backend == .stage2_c) return error.SkipZigTest;
     if (builtin.zig_backend == .stage2_x86_64) return error.SkipZigTest;
     if (builtin.zig_backend == .stage2_arm) return error.SkipZigTest;
```

Comment:
I wonder, do x64 and arm backends pass this as well? If not, any clue as to why aarch64 does without having SIMD support?

Reply:
Yes, they pass this test as well. I assume it's due to Sema knowing at comptime that the only possible of an `u0` is `0`.

Reply:
Mhm, makes sense!